workspaces/libnpmexec/lib/index.js

```diff
@@ -114,16 +140,9 @@ const exec = async (opts) => {
     args[0] = getBinFromManifest(manis[0])
   }
 
-  // figure out whether we need to install stuff, or if local is fine
-  const localArb = new Arborist({
-    ...flatOptions,
-    path,
-  })
-  const localTree = await localArb.loadActual()
-
-  // do we have all the packages in manifest list?
+  // are all packages from the manifest list installed?
   const needInstall =
-    manis.some(manifest => manifestMissing({ tree: localTree, manifest }))
+    manis.some(manifest => !manifest[_localManifest])
```

Comment:
isn't `manifest` potentially `undefined` here? won't that cause the prop check to explode?